### PR TITLE
Feature/openapi

### PR DESCRIPTION
**Pull request by AllenShen**
Signed-off-by: 沈冬冬 <sddhbk@gmail.com>

add description for changes of openapi

---
This change is [Reviewable](https://reviewable.io/reviews/koderover/zadig-doc/196)

### CODE REVIEW

File: dev/12.开发者中心/02.workflow-api.md [hunk 1]
```diff
@@ -248,3 +263,129 @@ POST /api/directory/workflowTask/create
...
+#### 返回说明
+
+|参数名|类型|描述|
+|---|---|---|
+|`workflow_name`|string|工作流名称|
+|`env_name`|string|环境名称|
+|`status`|string|任务状态|
+|`targets`|[][TargetArgs](#target_args)|构建参数|
+|`images`|[][Image](#image)|分发镜像信息|
+|`test_reports`|[][TestReport](#test_reports)|工作流执行完成回调信息
```
Comment (fansi404) [inline dev/12.开发者中心/02.workflow-api.md:354]: 这个不是测试报告信息么？

File: dev/12.开发者中心/02.workflow-api.md [hunk 2]
```diff
@@ -248,3 +263,129 @@ POST /api/directory/workflowTask/create
...
+
+#### 返回说明
+
+|参数名|类型|描述|
+|---|---|---|
+|`workflow_name`|string|工作流名称|
+|`env_name`|string|环境名称|
+|`status`|string|任务状态|
+|`targets`|[][TargetArgs](#target_args)|构建参数|
+|`images`|[][Image](#image)|分发镜像信息|
```
Comment (fansi404) [inline dev/12.开发者中心/02.workflow-api.md:353]: 字段和昨晚 Andrew 支持的接口保持一致？都用 release_images?

File: dev/12.开发者中心/02.workflow-api.md [hunk 3]
```diff
@@ -248,3 +263,129 @@ POST /api/directory/workflowTask/create
...
+|`test_reports`|[][TestReport](#test_reports)|工作流执行完成回调信息
+
+<h5 id="image">Image 参数说明</h5>
+
+|参数名|类型|描述|
+|---|---|---|
+|`image`|string|镜像地址|
+|`service_name`|string|服务名称|
```
Comment (fansi404) [inline dev/12.开发者中心/02.workflow-api.md:361]: `service_name` 还是 `service_module`？需要对齐吗？

File: dev/12.开发者中心/02.workflow-api.md [hunk 4]
```diff
@@ -248,3 +263,129 @@ POST /api/directory/workflowTask/create
...
+|参数名|类型|描述|
+|---|---|---|
+|`test_name`|string|测试名称|
+|`function_test_report`|[FunctionTestReport](#function_test_report)|功能测试报告概览|
+
+<h5 id="function_test_report">FunctionTestReport 参数说明 </h5>
+
+|参数名|类型|描述|
+|---|---|---|
+|`tests`|string|测试用例数量|
```
Comment (lilianzhu) [inline dev/12.开发者中心/02.workflow-api.md:375]: int

**Review by lilianzhu — APPROVED** [review]

**Pull request merged**